### ISSUE TITLE
lerna 3.16.0 fails on Windows for istanbuljs

### ISSUE THREAD

**Issue by coreyfarrell** [lerna/lerna#2189]
## Current Behavior
`lerna bootstrap --hoist` fails under Travis-CI for Windows for istanbuljs, see https://github.com/istanbuljs/istanbuljs/issues/447.

Log at https://travis-ci.org/istanbuljs/istanbuljs/jobs/560765659

<details><summary>lerna.json</summary><p>

```json
{
    "lerna": "3.10.3",
    "packages": ["packages/*"],
    "version": "independent"
}
```
</p></details>

## Your Environment
Windows for Travis-CI (I do not have other access to Windows)

| Executable | Version |
| ---: | :--- |
| `lerna --version` | 3.16.0 |
| `npm --version`  | 6.9.0 |
| `node --version` | 10.16.0 |

**Comment by evocateur** [comment]
Released in [v3.16.1](https://github.com/lerna/lerna/releases/tag/v3.16.1).

**Comment by evocateur** [comment]
Relaxed version range (with new minimum) in [v3.16.2](https://github.com/lerna/lerna/releases/tag/v3.16.2).
